controllers/authcontroller.ts

```diff
@@ -1,5 +1,5 @@
-import { Router } from 'https://deno.land/x/opine@0.12.0/mod.ts';
-import { config } from "https://deno.land/x/dotenv/mod.ts";
+import { Router } from 'https://deno.land/x/opine@1.9.0/mod.ts';
```

Comment:
I'm having trouble running these changes, but it works for me using the latest opine. Can you please change the dependency version for opine to `opine@2.0.0` for all places where we import opine?